README.md

```diff
@@ -11,11 +11,23 @@ There are actually multiple ways to run the CFE.
 The BMI functionality was developed as a standalone module in C. To compile this code the developer used these steps:
```

Comment:
typos (line 6): 1. As written by the ~~origional~~ original author. This ~~include~~ includes a full program...